server/seed-dev.sh

```diff
@@ -13,10 +13,6 @@ export PGPASSWORD=$SQL_PASSWORD
 read -r -p "Are you sure? This will delete existing data in your local '$SQL_DATABASE' database. [y/N] " response
 case "$response" in
   [yY][eE][sS] | [yY])
-    if [[ "$(database_exists)" ]]; then
-      drop_database
```

Comment:
It was convenient for the script to drop the DB so I could start all over with a new matching schema. It looks like you removed this convenience. Am I mistaken?

Reply:
This is removed in favor of using containerized DB (I had a discussion with the team on this during a s/u I think you may have missed). The way this is originally written causes issues if connections are left open etc. The new way of doing things is to use the command `yarn stop-db` which stops the DB container and wipes the database data. Let me know if you have a use case for keeping this though.